cmd/publish-release/cmd/github.go

```diff
@@ -166,26 +177,102 @@ func init() {
 	rootCmd.AddCommand(githubPageCmd)
 }
 
-func getAssetsFromStrings(assetStrings []string) []announce.Asset {
+func getAssetsFromStrings(assetStrings []string) ([]announce.Asset, error) {
 	r := []announce.Asset{}
+	tempFiles := []string{}
```

Comment:
this is not used will be empty, do we need that?

Reply:
Forgot to collect the garbage. Done.